assets/style/style.css

```diff
@@ -0,0 +1,2836 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+  width: 100vw;
+}
+
+.top-header {
+  position: fixed;
+  z-index: 1;
+}
+
+a {
+  text-decoration: none;
+}
+
+.white-nav a,
+.top-div li,
+.top-div a,
+.white-nav li {
+  color: #808080;
+  font-size: 12px;
+  list-style-type: none;
+  padding: 5px;
+  text-decoration-line: none;
+}
+
+.white-nav a,
+.white-nav li {
+  color: #999595;
+  font-size: 13px;
+  line-height: 10px;
+  list-style-type: none;
+  padding: 5px;
+  text-decoration-line: none;
+}
+
+#news {
+  color: #fff;
+  margin-left: 10px;
+}
+
+.white-cont {
+  position: sticky;
+  z-index: -1;
+}
+
+.top-nav {
+  background-color: #1f1d1d;
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  justify-content: space-between;
+}
+
+.left-list {
+  display: grid;
+  grid-template-columns: repeat(6, auto);
+  width: 500px;
+}
+
+.right-list {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  width: 300px;
+}
+
+.event-list,
+.bizlist {
+  display: none;
+  position: absolute;
+}
+
+.event-drop:hover > .event-list,
+.biz-drop:hover > .bizlist {
+  background-color: #000;
+}
+
+#neural {
+  margin-right: 80px;
+}
+
+.info0 a {
+  color: #f42;
+  font-size: 15px;
+  font-weight: 700;
+  margin-bottom: 20px;
+  position: relative;
+  right: 5px;
+  text-decoration: none;
+}
+
+.yellow a {
+  color: #251f1f;
+  font-size: 12px;
+  margin-left: 20px;
+  position: relative;
+  text-decoration-line: none;
+  top: 5px;
+}
+
+.right-art-top a {
+  color: #fff;
+  font-size: 25px;
+  left: 15px;
+  line-height: 30px;
+  position: relative;
+  text-decoration: none;
+  top: 20px;
+  z-index: 0;
+}
+
+.img-clue a {
+  color: #000;
+  font-size: 16px;
+  text-justify: auto;
+  word-wrap: break-word;
+}
+
+.reddit-div a {
+  color: #fff;
+  font-size: 14px;
+  left: 20px;
+  position: relative;
+  top: 80px;
+  z-index: 0;
+}
+
+.funding-list a {
+  color: #f42;
+}
+
+.latest-div4 a {
+  bottom: 0;
+  color: #f00;
+  position: relative;
+  left: 10px;
+}
+
+.red-txt a {
+  color: #f42;
+  font-size: 12px;
+  width: 200px;
+}
+
+.latest-div1 a {
+  bottom: 25px;
+  color: #f42;
+  position: relative;
+  width: 200px;
+  font-size: 12px;
+}
+
+.latest-div2 a {
+  bottom: 0;
+  color: #f42;
+  font-size: 15px;
+  position: relative;
+  width: 200px;
+}
+
+.latest-div3 a {
+  bottom: 0;
+  color: #f42;
+  font-size: 12px;
+  padding-top: 10px;
+  position: relative;
+  width: 200px;
+  left: 10px;
+}
+
+.apps-anch a {
+  font-size: 15px;
+  line-height: 15px;
+  color: #797373;
+}
+
+.latest-div5 a {
+  color: #f42;
+  padding-top: 10px;
+  width: 200px;
+  position: relative;
+  bottom: 0;
+  font-size: 12px;
+  left: 10px;
+}
+
+.latest-div6 a {
+  bottom: 10px;
+  color: #f42;
+  font-size: 11px;
+  left: 10px;
+  padding-top: 10px;
+  position: relative;
+  width: 200px;
+  top: 10px;
+}
+
+.latest-div7 a {
+  bottom: 15px !important;
```

Comment:
### please remove all the !important keywords and try to override style based on CSS selector specificity, here is a good [article](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) talking about it